frontend/web/styles/3rdParty/_bootstrap.scss

```
@@ -16,6 +16,40 @@
 // 4. Optionally include utilities API last to generate classes based on the Sass map in `_utilities.scss`
 
 @import "../project/spacing-utils";
+
+// Colour is owned by the design-system token utilities (_token-utilities.scss).
+// Drop the overlapping keys from Bootstrap's text-colour utility so `.text-*`
+// resolve to the tokens, not Bootstrap's `$theme-colors`. Otherwise Bootstrap's
+// `!important` copy wins the cascade and the token utility is dead.
+//
```

Stylelint (17.14.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)